assets/components/odf-lvm/topolvm_default-storage-class.yaml

```diff
@@ -1,9 +1,13 @@
 apiVersion: storage.k8s.io/v1
 kind: StorageClass
 metadata:
-  name: kubevirt-hostpath-provisioner
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
-provisioner: kubevirt.io/hostpath-provisioner
+  labels:
+  name: topolvm-provisioner
+parameters:
+  csi.storage.k8s.io/fstype: xfs
```

Comment:
Is this something users will eventually want to configure here or would we suggest that they create a separate StorageClass to do that?

Reply:
The storageClass is typically a cluster-admin defined API, so it's reasonable to assume to microshift users would want to specify this.

Reply:
OK, so we don't need to provide a config option for this, but we might need to give documentation for how to create an alternative storage class.

Reply:
+1

Reply:
:+1